src/main/java/com/assignment/Assignment5.java

```diff
@@ -0,0 +1,12 @@
+package com.assignment;
+
+import java.util.Arrays;
+
+public class Assignment5 {
+    public static String ronaldo() {
+        //Find the index of Ronaldo in the given array
+        String [] name = {"Paulo Dybala","Federico","Gianluigi","Ronaldo","Messi"};
+        String a = Integer.toString(Arrays.asList(name).indexOf("Ronaldo"));
+        return a;
+    }
+}
```

Comment:
Please find my below comments :

The name for the function is not a meaningful one.
Any reason why we we are defining this as a static function?
This is not an optimised approach. Lets create a generic function instead of hardcoding the strings :
String getIndexFromStringList(String stringList, String string)